src/app/components/login/login.component.html

```diff
@@ -0,0 +1,40 @@
+<div class="container vh-100" style="max-width: 400px;"> 
+    <div class="row justify-content-center h-100">
+        <div class="card my-auto shadow">
+            <div class="card-body px-5">
+                <div class="text-center">
+                    
+                </div>
+                <div class="text-center">
+                    <h2>Welcome</h2>
+                </div>
+                <p style="color: lightslategray; text-align: center;">
+                    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Please enter your login
```

Comment:
This is not the ideal way. Do find alternative solution to give padding or margin